product-service/lambda-functions/createProduct.ts

```diff
@@ -0,0 +1,108 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import * as AWS from "aws-sdk";
+import { v4 as uuidv4 } from "uuid";
+import { IProduct } from "./product.interface";
+
+const dynamoDb = new AWS.DynamoDB.DocumentClient();
+const PRODUCTS_TABLE_NAME = process.env.PRODUCTS_TABLE_NAME || "";
+const STOCKS_TABLE_NAME = process.env.STOCKS_TABLE_NAME || "";
+
+const validateProduct = ({
+  title,
+  price,
+  count,
+}: Omit<IProduct, "id" | "description"> & { count: number }): string | null => {
+  if (
+    typeof price !== "number" ||
+    price < 0 ||
+    Number.isInteger(count) ||
```

Comment:
what a terrible little bug
```suggestion
    !Number.isInteger(count) ||
```